components/DynamicComponent.jsx

```diff
@@ -0,0 +1,18 @@
+import dynamic from "next/dynamic";
```

Comment:
maybe call this <renderCustomElement> instead? it should be focussed on custom elements, not any kind of dynamic compoenent